pkg/manager/member/pd_upgrader.go

```diff
@@ -14,19 +14,160 @@
 package member
 
 import (
+	"fmt"
+	"sort"
+
 	"github.com/pingcap/tidb-operator/pkg/apis/pingcap.com/v1alpha1"
+	"github.com/pingcap/tidb-operator/pkg/controller"
+	"github.com/pingcap/tidb-operator/pkg/label"
 	apps "k8s.io/api/apps/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	corelisters "k8s.io/client-go/listers/core/v1"
 )
 
-type pdUpgrader struct{}
+const (
+	// ImagePullBackOff is the pod state of image pull failed
+	ImagePullBackOff = "ImagePullBackOff"
```

Comment:
Can we import these consts from K8s's code base?

Reply:
these consts does not defined in K8s

Reply:
https://github.com/kubernetes/kubernetes/blob/7f23a743e8c23ac6489340bbb34fa6f1d392db9d/pkg/kubelet/images/types.go#L27